### PR TITLE
Update dev env to centos:8

### PR DESCRIPTION
**Pull request by shanemcd**

### CODE REVIEW

File: tools/docker-compose/Dockerfile [hunk 1]
```diff
@@ -47,24 +44,25 @@ RUN yum -y install acl \
...
-  python-devel \
-  python-pip \
-  python-psutil \
-  python-psycopg2 \
-  python-setuptools \
-  python36-devel \
-  python36-setuptools \
+  @postgresql:10 \
+  postgresql-devel \
+  python2 \
```
Comment (wenottingham) [inline tools/docker-compose/Dockerfile]: what for?

Comment (shanemcd) [reply tools/docker-compose/Dockerfile]: oops. you're right.

Comment (ryanpetrello) [reply tools/docker-compose/Dockerfile]: ^

File: tools/docker-compose/Dockerfile [hunk 2]
```diff
@@ -117,23 +113,15 @@ RUN cd /tmp && VENV_BASE="/venv" make requirements_dev
...
 RUN n 10.15.0
 
-# Install a newer SQLite to make Django happy (centos7 has a really old one,
-# and we can probably stop doing this in centos8)
-RUN curl -O "https://sqlite.org/2019/sqlite-autoconf-3290000.tar.gz"
-RUN tar zxvf sqlite-autoconf-3290000.tar.gz
-RUN cd sqlite-autoconf-3290000 && ./configure && make && make install
-RUN mv -f /usr/local/lib/libsqlite3.so.0 /lib64/
-RUN mv -f /usr/local/lib/libsqlite3.so.0.8.6 /lib64/
-
```
Comment (wenottingham) [inline tools/docker-compose/Dockerfile:127]: :tada: :tada: :tada:

File: Makefile [hunk 3]
```diff
@@ -196,7 +196,7 @@ requirements_awx_dev:
 
 requirements: requirements_ansible requirements_awx
 
-requirements_dev: requirements requirements_awx_dev requirements_ansible_dev
+requirements_dev: requirements_awx requirements_ansible_py3 requirements_awx_dev requirements_ansible_dev
```
Comment (ryanpetrello) [inline Makefile:199]: 🔪 🐍 🥈

File: tools/docker-compose/Dockerfile (hunk 2 above)
Comment (ryanpetrello) [reply tools/docker-compose/Dockerfile:127]: god yes

**Review by ryanpetrello — APPROVED** [review]

**Pull request merged**